### PR TITLE
Ribbon covers hamburger menu fixes #359

### LINKED ISSUE #359: Ribbon covers the hamburger menu and responsiveness fix
image

Issue approved by @ananya-agrawal

### PR DESCRIPTION
**Pull request by anupm12**
* Please fill out the template below

### Did you read the instructions in [README](../README.md) ?

Yes

### Solves which issue ?

Fixes #359 

### Give discription.

image

## Checklist

(To mark the checkboxes use `x` within the box brackets)

- [x] Signed the code of conduct.

- [x] Have you added yourself to the participants.

- [x] Joined opencode zulip server.

### CODE REVIEW

**Comment by anupm12** [comment]
Live link 
https://ribbon-issue.netlify.com/

**Comment by anupm12** [comment]
Ok @prateek76 will do it asap

**Comment by anupm12** [comment]
I'm sorry
I mistakenly clicked on close button 😅

**Comment by prateek76** [comment]
@anupm12 fix responsiveness of page

**Comment by anupm12** [comment]
@prateek76 I think that problem is in master branch.
Although I tried to solve but could not,
Even if I inspect nothing is shown in that blank space.

**Comment by anupm12** [comment]
image

**Comment by prateek76** [comment]
@anupm12  
ok if there is problem in master try to fix it i have updated the issue

this will help try finding width of elements set to size in px

capture

**Comment by anupm12** [comment]
@prateek76 ok

**Comment by anupm12** [comment]
@prateek76 I did solve that wide white space and now there is a small vertical scroll 😅 
image

**Comment by anupm12** [comment]
@prateek76 
I pushed the changes.
live link: https://ribbon-issue.netlify.com/

Was unable to understand the opening and closing tags so I did beautify index.html file.
Sorry for that.

**Comment by prateek76** [comment]
> @prateek76
> I pushed the changes.
> live link: https://ribbon-issue.netlify.com/
> 
> Was unable to understand the opening and closing tags so I did beautify index.html file.
> Sorry for that.

push it

**Comment by anupm12** [comment]
@prateek76 Getting some error.
Give me few minutes

**Comment by prateek76** [comment]
> @prateek76 Getting some error.
> Give me few minutes

you can take hours 😄 no problem

File: index.html [hunk 1]
```diff
@@ -1014,48 +1034,47 @@ <h2 class="my-4">Issues</h2>
...
-// When the user scrolls the page, execute myFunction 
-window.onscroll = function() {myFunction()};
-  
   function myFunction() {
     var winScroll = document.body.scrollTop || document.documentElement.scrollTop;
     var height = document.documentElement.scrollHeight - document.documentElement.clientHeight;
     var scrolled = (winScroll / height) * 100;
     document.getElementById("myBar").style.width = scrolled + "%";
   }
+<<<<<<< HEAD
```
Comment (prateek76) [inline index.html]: here

File: index.html [hunk 2]
```diff
@@ -1071,5 +1090,6 @@ <h2 class="my-4">Issues</h2>
     document.documentElement.scrollTop = 0;
   }
   </script>
+>>>>>>> 9601cdb6a243823b98ab0ce4cb8b81206acf2788
```
Comment (prateek76) [inline index.html]: .

**Comment by anupm12** [comment]
@prateek76 I did

**Comment by anupm12** [comment]
ok @prateek76

**Comment by anupm12** [comment]
@prateek76 Will I get points for two issues?
One is this and other which you mentioned vertical scrolling one 😅

**Pull request merged**